tracer/build/_build/Build.Steps.cs

```diff
@@ -978,7 +893,7 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                 };
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 888 to 889]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Don't we want symlinks into `{arch}/` instead of the actual files?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/DataDog/dd-trace-dotnet/5775?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Oh, I see it's handled below, at least for musl artifacts.